source/_integrations/monoprice.markdown

```diff
@@ -55,3 +55,67 @@ Restore a previously taken snapshot of one or more speakers. If `entity_id` is `
 | Service data attribute | Optional | Description |
 | ---------------------- | -------- | ----------- |
 | `entity_id` | yes | String or list of strings that point at `entity_id`s of zones.
+
+
+## Examples
+
+### Script
+
+```yaml
+# Take Snapshot of current state, turn on zones, select source.
+alias: Notify - Media Announment
+sequence:
+  - service: monoprice.snapshot
+    data: {}
+    target:
+      entity_id:
+        - media_player.basement
+        - media_player.garage
+        - media_player.great_room
+        - media_player.kitchen
+        - media_player.upstairs_master
+  - service: media_player.turn_on
+    data: {}
+    target:
+      entity_id:
+        - media_player.kitchen
+        - media_player.great_room
+        - media_player.garage
+        - media_player.upstairs_master
+        - media_player.basement
+  - service: media_player.select_source
+    data:
+      source: Server
+    target:
+      entity_id:
+        - media_player.basement
+        - media_player.kitchen
+        - media_player.upstairs_master
+        - media_player.great_room
+        - media_player.garage
+mode: single
+icon: mdi:speaker-message
+```
+
+### Automations
+
+```yaml
+alias: Notify - Doorbell
```

Comment:
I'm not really sure why a separate script is used in this case; it makes the more complicated than it needs to be.

I would suggest making it single automation instead.

Reply:
Done.  I used a script in mine so I could reuse it for different announcements / automations.  I've simplified it for the example per your suggestion.